src/mscorlib/src/System/AggregateException.cs

```diff
@@ -29,7 +29,9 @@ namespace System
     /// <see cref="AggregateException"/> is used to consolidate multiple failures into a single, throwable
     /// exception object.
     /// </remarks>
+#if FEATURE_SERIALIZATION
     [Serializable]
```

Comment:
They are, actually.  [Serializable] is a pseudo attribute; the C# compiler sees the attribute but doesn't actually add it as a custom attribute, instead marking the type as serializable in special metadata.  And it does so based on the attribute's name, so any time marked with either this SerializableAttribute or the SerializableAttribute in corefx will work.  The same applies to [NonSerializable].